### PR TITLE
video / pdf page desktop header fix

### PR DESCRIPTION
**Pull request by gumaerc**
#### Pre-Flight checklist

- [x] Screenshots and design review for any changes that affect layout or styling
  - [x] Desktop screenshots
  - [x] Tag @ferdi or @pdpinch for review
- [x] Testing
  - [x] Code is tested
  - [x] Changes have been manually tested

#### What are the relevant tickets?
None

#### What's this PR do?
Fixes a bug where the header does not display properly on video and pdf pages

#### How should this be manually tested?
Run the site and visit the deploy preview, ensure that the header is being rendered properly.  Here are a few examples:

http://localhost:3000/courses/ec-711-d-lab-energy-spring-2011/sections/intro-energy-basics-human-power/mitec_711s11_read_react/
http://localhost:3000/courses/ec-711-d-lab-energy-spring-2011/sections/week-7-trip-planning-and-preparations/lecture-7-solar-cookers-creative-capacity-building-trip-preparation/

#### Screenshots (if appropriate)
image

### CODE REVIEW

File: site/layouts/course/pdf.html [hunk 1]
```diff
@@ -1,3 +1,10 @@
+{{ define "header" }}
+{{ partial "header" . }}
+{{ partial "header_placeholder.html" . }}
+{{ end }}
+{{ define "subheader" }}
+{{ partial "chp_partial.html" (dict "partial" "course_banner.html" "context" .) }}
+{{ end }}
```
Comment (aliceriot) [inline site/layouts/course/pdf.html:7]: can we just change this in the course `baseof.html` template instead? I just added this line to `course/baseof.html` and the header is on all course pages:

```
    {{ block "header" . }}{{ partial "header" . }}{{ partial "header_placeholder.html" . }}{{ end }}
```

**Review by aliceriot — CHANGES_REQUESTED** [review]

**Comment by aliceriot** [comment]
also it looks like there's some other changes - did you mean to put those in this branch?

**Comment by gumaerc** [comment]
@aliceriot should be all set now

File: site/layouts/partials/course_content.html [hunk 2]
```diff
@@ -3,10 +3,12 @@
...
-    <h1 class="text-uppercase font-weight-bold">{{ $currentPage.Title }}</h1>
-    {{ with $currentPage.Params.course_info.subtitle }}
-    <span class="subtitle">{{ . }}</span>
-    {{ end }}
+    <div class="course-section-title-container">
+      <h1 class="text-uppercase font-weight-bold">{{ $currentPage.Title }}</h1>
+      {{ with $currentPage.Params.course_info.subtitle }}
+      <span class="subtitle">{{ . }}</span>
+      {{ end }}
+    </div>
```
Comment (aliceriot) [inline site/layouts/partials/course_content.html:11]: was this change accidentally included from https://github.com/mitodl/hugo-course-publisher/pull/364/?

Comment (gumaerc) [reply site/layouts/partials/course_content.html:11]: @aliceriot I accidentally created this PR from the wrong branch, I'm just going to recreate it as that will be easier.

**Comment by gumaerc** [comment]
re-creating this PR as it's based off the wrong branch

**Pull request closed without merging**